Makefile

```
@@ -118,11 +118,11 @@ test_images:
 
 .PHONY: tidy
 tidy:
-	go mod tidy -go 1.20
+	go mod tidy -go 1.22
```

```suggestion
	go mod tidy
```

It's already in `go.mod`

---

That makes sense.
Done.